### ISSUE TITLE
hub does not work with "--literal-pathspecs" option

### ISSUE THREAD

**Issue by DarwinAwardWinner** [github/hub#1018]
Hub just acts like regular git when the `--literal-pathspecs` option is used. This is a problem if one wants to use hub with [magit](https://github.com/magit/magit), which uses this option. 

```
# Run in a git repo with a github remote
$ hub issue; echo $?
0
$ hub --literal-pathspecs issue; echo $?
git: 'issue' is not a git command. See 'git --help'.
1
```

My uninformed guess is that the `--literal-pathspecs` option was recently added to git and hub doesn't know about it, and on encountering an unknown argument hub enters a pass-through mode for compatibility.

**Comment by mislav** [comment]
This was fixed in #951 and slated for a release soon!

Thanks for reporting.

**Comment by DarwinAwardWinner** [comment]
Ok, I guess I'll just install from source to get the fix for now.
